docs/source/getting_started.rst

```diff
@@ -0,0 +1,171 @@
+Getting started
+===============
+
+For installation of the Ankaios SDK, see the `Installation section <index.html#installation>`_.
+
+Once the SDK is installed, you can start using it by importing the module and creating an ankaios object.
+
+.. code-block:: python
+
+    from ankaios import Ankaios
+
+    ankaios = Ankaios()
+
+The initialization of the Ankaios object will automatically connect to the fifo pipes of the control interface. Once this is done,
+the communication with the Ankaios ecosystem can be started.
+
+**Apply a manifest**
+--------------------
+
+Considering we have a manifest file with a workload called ``nginx_test`` and a config called ``test_ports``. The manifest file is as follows:
+
+.. code-block:: yaml
+    :caption: my_manifest.yaml
+
+    apiVersion: v0.1
+    workloads:
+        nginx_test:
+            runtime: podman
+            restartPolicy: NEVER
+            agent: agent_A
+            configs:
+                ports: test_ports
+            runtimeConfig: |
+                image: image/test
```

Comment:
The `ports` config is not used in the runtimeConfig. Maybe we should omit the configs here to have a minimal example that is easy to understand.

Reply:
Alright. Done.